apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -133,22 +133,58 @@ const getNativeElementsList = (elements: SBEnumType['value']): JSX.Element => {
   );
 };
 
-const RenderArgsTable = ({ hideArgsTable, primaryStory }: { primaryStory: PrimaryStory; hideArgsTable: boolean }) => {
+const slotRegex = /as\?:\s*"([^"]+)"/;
```

Comment:
it seems this is not catching all Slot apis as expected for example (Avatar Component / https://react.fluentui.dev/?path=/docs/components-avatar--docs):

![image](https://github.com/user-attachments/assets/dadfeb3c-35e8-4543-8e75-437a77554b0c)

**Todo:**

- [ ] make sure we are catching all actual Slot definitions